src/layers/RouteLayer/RouteLayer.js

```diff
@@ -157,7 +158,8 @@ class RouteLayer extends CasaLayer {
           ),
         });
         feature.setProperties(sequenceProps);
-        if (finalDestination) feature.set('final', finalDestination);
+        if (isStart) feature.set('start', isStart);
```

Comment:
Always use brackets in if clauses.
Here, you don't need an `if`:

```
feature.setProperties({
  ...sequenceProps,
  isStart, // 1 variable name is enough (not `isStart` and `start`)
  isEnd,
});

Reply:
I agree it makes it more readable, but this way we set unnecessary properties for all features (now ALL features receive both properties, isStart and isEnd), inflating the data slightly. But I guess we don't care